### PR TITLE
ALE 0.9.0 ticks

### PR DESCRIPTION
**Pull request by acpaquette**

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Probably worth getting a reviewer that isn't just me on this. 

I didn't include a changelog in my PR... should we add that here or should I open a different one?

**Review by amystamile-usgs — APPROVED** [review]

File: CHANGELOG.md
Comment (jlaura) [inline CHANGELOG.md:1]: Need to bump the version in line 36?

File: code.json [hunk 1]
```diff
@@ -42,6 +42,52 @@
...
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-04-07"
+    }
+  }, {
+    "name": "ale",
+    "organization": "U.S. Geological Survey",
+    "description": "GitLab repository for the Abstraction Layer for Ephemerides package",
+    "version": "0.8.8",
```
Comment (jlaura) [inline code.json:52]: 0.9.0? Based on the tagged releases, it looks like this should have a 0.8.8 and a 0.9.0 entry?

File: code.json [hunk 2]
```diff
@@ -42,6 +42,52 @@
...
+    ],
+
+    "languages": [
+      "C++",
+      "Python"
+    ],
+
+    "contact": {
+      "name": "Jason Laura",
+      "email": "jlaura@usgs.gov"
```
Comment (jlaura) [inline code.json:88]: This should be you, no? I was just in the last one because I did this part of the release.

File: code.json [hunk 3]
```diff
@@ -42,6 +42,52 @@
...
+    },
+
     "date": {
       "metadataLastUpdated": "2023-02-23"
```
Comment (jlaura) [inline code.json:92]: Bump date?

**Review by jlaura — APPROVED** [review]

File: code.json [hunk 4]
```diff
@@ -38,12 +38,106 @@
...
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://code.usgs.gov/astrogeology/ale/",
+    "downloadURL": "https://code.usgs.gov/astrogeology/ale/-/archive/main/ale-main.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/DISCLAIMER.md",
```
Comment (jlaura) [inline code.json:68]: These can't aim at main and have a 0.9.0 version. The version has to aim to where the artifact will be.

**Review by jlaura — CHANGES_REQUESTED** [review]

File: code.json [hunk 5]
```diff
@@ -58,14 +58,14 @@
       "licenses": [
         {
           "name": "Public Domain, CC0-1.0",
-          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/LICENSE.md"
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/0.9/LICENSE.md"
```
Comment (jlaura) [inline code.json:61]: 0.9.0, not 0.9

**Review by jlaura — CHANGES_REQUESTED** [review]

File: code.json (hunk 5 above)
Comment (acpaquette) [reply code.json:61]: So the branch will be 0.9, with bugfix versions being tags off of the 0.9 branch

Comment (jlaura) [reply code.json:61]: I'm saying, I think you need the bug fix '.0' at the end of the tag. Unless you want to have:
- 0.8.8
- 0.9
- 0.9.1
- 0.9.2
- 0.9.x
- 0.10
- 0.10.1

File: README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
 [![Build Status](https://travis-ci.org/USGS-Astrogeology/ale.svg?branch=master)](https://travis-ci.org/USGS-Astrogeology/ale)
 [![Coverage Status](https://coveralls.io/repos/github/USGS-Astrogeology/ale/badge.svg?branch=master)](https://coveralls.io/github/USGS-Astrogeology/ale?branch=master)
 [![Docs](https://readthedocs.org/projects/ale/badge/?version=latest)](https://ale.readthedocs.io/en/latest/?badge=latest)
-
+[![DOI](https://img.shields.io/badge/DOI-10.5066/P906D84L-blue)](https://www1.usgs.gov/csas/doi/#/form/doi:10.5066%2FP906D84L)
```
Comment (jlaura) [inline README.md:9]: so pretty

File: code.json (hunk 5 above)
Comment (jlaura) [reply code.json:61]: The docker code check will fail here. You have to be consistent with the "version":"0.9.0" and the URLS in line 61, 67, 68. They all have to be the same version number (0.9 or 0.9.0) - no mixing).

**Review by jlaura — APPROVED** [review]

**Review by jlaura — APPROVED** [review]

**Pull request merged**